projects/pgai/tests/semantic_catalog/test_builder.py

```diff
@@ -13,16 +13,778 @@
     "boarding_pass",
     "booking",
     "booking_leg",
+    "events",
     "flight",
     "frequent_flyer",
     "passenger",
     "phone",
 }
 
-ALL_VIEWS: set[str] = {"flight_summary", "passenger_details"}
+ALL_VIEWS: set[str] = {"events_daily", "flight_summary", "passenger_details"}
 
 ALL_PROCS: set[str] = {"advance_air_time", "update_flight_status"}
 
+TIMESCALE_PROCS: set[str] = {
```

Comment:
I think this is fine for now, but we may want to revisit so we aren't chasing changes to extensions that install stuff in the public schema. Maybe we should create an "empty" schema to use instead of public for these tests.